### PR TITLE
Cw behat mo feedback function name and doc (global)

### PR DESCRIPTION
**Pull request by mike1937**
Global Sample feedback

### CODE REVIEW

File: Sample_Files/contexts/LoginContext.php [hunk 1]
```diff
@@ -3,10 +3,22 @@
 class LoginContext extends PageContext  {
 
   /**
+   * Login Page instance.
+   *
```
Comment (mike1937) [inline Sample_Files/contexts/LoginContext.php:7]: Sample for description on property

File: Sample_Files/contexts/LoginContext.php [hunk 2]
```diff
@@ -3,10 +3,22 @@
...
    * @var LoginPage
    */
   private $login_page;
 
+  // Fields.
+  const FIELD_USERNAME = 'USERNAME';
```
Comment (mike1937) [inline Sample_Files/contexts/LoginContext.php:13]: Sample of using const. see self::FIELD_USERNAME in line 46 and others

File: Sample_Files/pages/LoginPage.php [hunk 3]
```diff
@@ -1,5 +1,12 @@
 <?php
+/**
+ * @file
+ * Login page file.
+ */
```
Comment (mike1937) [inline Sample_Files/pages/LoginPage.php:5]: Sample of @file on all files

File: Sample_Files/pages/LoginPage.php [hunk 4]
```diff
@@ -1,5 +1,12 @@
...
 
+/**
+ * Class LoginPage implements the behavior for login pages.
+ */
```
Comment (mike1937) [inline Sample_Files/pages/LoginPage.php:9]: Sample of description for classes

File: Sample_Files/pages/LoginPage.php [hunk 5]
```diff
@@ -43,9 +50,10 @@ class LoginPage extends Page {
 
   /**
    * The path.
+   *
    * @return string
    */
-  public function get_path() {
+  public function getPath() {
```
Comment (mike1937) [inline Sample_Files/pages/LoginPage.php:56]: Sample for function names within classes

File: Sample_Files/pages/LoginPage.php [hunk 6]
```diff
@@ -58,7 +66,11 @@ public function get_all_fields() {
   }
 
   /**
-   * A specific field.
+   * Gets a field.
+   *
+   * @param string $field
```
Comment (mike1937) [inline Sample_Files/pages/LoginPage.php:71]: Sample for @param

File: Sample_Files/pages/LoginPage.php [hunk 7]
```diff
@@ -43,9 +50,10 @@ class LoginPage extends Page {
 
   /**
    * The path.
+   *
```
Comment (mike1937) [inline Sample_Files/pages/LoginPage.php:53]: Sample for using space before "@return"

File: Sample_Files/pages/LoginPage.php (hunk 5 above)
Comment (brendanmacdonald) [inline Sample_Files/pages/LoginPage.php:56]: Updating functions names across all .php files.

File: Sample_Files/contexts/LoginContext.php (hunk 1 above)
Comment (brendanmacdonald) [inline Sample_Files/contexts/LoginContext.php:7]: Updated all context files.

File: Sample_Files/contexts/LoginContext.php (hunk 2 above)
Comment (brendanmacdonald) [inline Sample_Files/contexts/LoginContext.php:13]: Updated all .php files to use consts.

File: Sample_Files/pages/LoginPage.php (hunk 3 above)
Comment (brendanmacdonald) [inline Sample_Files/pages/LoginPage.php:5]: @file added to all files.

File: Sample_Files/pages/LoginPage.php (hunk 4 above)
Comment (brendanmacdonald) [inline Sample_Files/pages/LoginPage.php:9]: Added to all classes.

File: Sample_Files/pages/LoginPage.php (hunk 6 above)
Comment (brendanmacdonald) [inline Sample_Files/pages/LoginPage.php:71]: Updated all return functions.

File: Sample_Files/pages/LoginPage.php (hunk 7 above)
Comment (brendanmacdonald) [inline Sample_Files/pages/LoginPage.php:53]: Added space in all @return functions.

**Pull request closed without merging**